### PR TITLE
ZEPPELIN-1484 Organize env.sh and site.xml config templates and docs

### PR DESCRIPTION
**Pull request by r-kamath**
### What is this PR for?

Organize env.sh and site.xml config templates and docs
### What type of PR is it?

Improvement
### What is the Jira issue?

ZEPPELIN-1484
### How should this be tested?

Read config templates and install doc
### Questions:
- Does the licenses files need update? n/a
- Is there breaking changes for older versions? n/a
- Does this needs documentation? yes (done)

### CODE REVIEW

File: conf/zeppelin-env.sh.template [hunk 1]
```diff
@@ -17,26 +17,56 @@
 #
 
 # export JAVA_HOME=
-# export MASTER=                 		# Spark master url. eg. spark://master_addr:7077. Leave empty if you want to use local mode.
-# export ZEPPELIN_JAVA_OPTS      		# Additional jvm options. for example, export ZEPPELIN_JAVA_OPTS="-Dspark.executor.memory=8g -Dspark.cores.max=16"
-# export ZEPPELIN_MEM            		# Zeppelin jvm mem options Default -Xmx1024m -XX:MaxPermSize=512m
-# export ZEPPELIN_INTP_MEM       		# zeppelin interpreter process jvm mem options.
-# export ZEPPELIN_INTP_JAVA_OPTS 		# zeppelin interpreter process jvm options.
+# export MASTER=                        # Spark master url. eg. spark://master_addr:7077. Leave empty if you want to use local mode.
```
Comment (corneadoug) [inline conf/zeppelin-env.sh.template]: The descriptions used to be all indented, but not anymore

Comment (r-kamath) [inline conf/zeppelin-env.sh.template]: you mean all # aligned or 8 spaces ?

Comment (corneadoug) [inline conf/zeppelin-env.sh.template]: All aligned would be better

Comment (r-kamath) [inline conf/zeppelin-env.sh.template]: cool

**Pull request closed without merging**